projects/laji/src/app/shared-modules/select/select-subcategories/select-subcategories.component.ts

```diff
@@ -302,7 +302,7 @@ export class SelectSubcategoriesComponent implements OnChanges {
       });
     }
 
-    this.open = (!this.open && Object.keys(this.selectedOptions).length > 0) ? true : this.open;
+    this.open = true;
```

Comment:
if this need to be always true. I would remove this line all together and change the line `
  [open]="open"` to `
  [open]="true"` in `metadata-select-with-subcategories.component.html`